### PR TITLE
Restaurant website

### PR DESCRIPTION
**Pull request by Leon-Mbegera**
A coffee restaurant page displaying the diverse types of brews served to customers, and a contact page for how to reach us.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "stylelint-scss": "^3.20.1",
+    "webpack": "^5.49.0",
+    "webpack-cli": "^4.7.2"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "watch": "webpack --watch"
+  },
```
Comment (george-shammar) [inline package.json:27]: [OPTIONAL]

- [ ] Please, consider add ` "start": "webpack-dev-server --mode development --open"` to your `package.json` scripts.

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+## Built with
+- Js
+- HTML
+- CSS & Bootstrap
+
+## To get started
+
+- Clone the repository and cd into the library project.
+- ` $ git clone https://github.com/Leon-Mbegera/Restaurant.git` 
+- ` $ cd Restaurant`
```
Comment (george-shammar) [inline README.md:14]: [OPTIONAL]

- [ ] You have a beautiful project, consider adding a screenshot to your README.

**Review by george-shammar — APPROVED** [review]
Hi @Leon-Mbegera , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Impressive :+1: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: package.json (hunk 1 above)
Comment (Leon-Mbegera) [reply package.json]: optional addition of code to my script done!

File: README.md (hunk 2 above)
Comment (Leon-Mbegera) [reply README.md]: Added restaurant home page screenshot to readme.

**Pull request merged**